.github/workflows/repeat_build.yml

```diff
@@ -20,7 +20,7 @@ on:
           - green_fast_track
           #- knownprojects
           - pluto
-          - ztl
```

Comment:
the `plan` command created in this job uses this string input, so needed to tweak it